src/icons/svg/arrow-down.svg

```diff
@@ -1,3 +1,3 @@
 <!--category:navigation-->
 <!--character:E0A0-->
-<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M15.9752 16.3512h-2.5217a.2075.2075 0 0 1-.2083-.2067V3.25c0-.6908-.56-1.25-1.25-1.25-.691 0-1.2501.5592-1.2501 1.25V16.143a.2084.2084 0 0 1-.2083.2083H8.0134c-.46 0-.8334.3734-.8334.8342 0 .22.0875.4317.2442.5883l3.981 3.9818c.325.3259.8524.3259 1.1783.0008l3.9826-3.9826a.8345.8345 0 0 0 .1791-.9083.8347.8347 0 0 0-.77-.5142z"/></svg>
\ No newline at end of file
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path id="M11.892869 5.807012L12.005763 5.8c.45882 0 .837451.343337.892987.787106L12.905763 6.7 12.905 15.093l2.418991-2.40993c.307612-.261325.755182-.285883 1.089679-.05979l.096703.075362c.340287.303999.395015.80744.145658 1.17442l-.07556.09645-3.864952 3.933012c-.330288.367777-.888393.396997-1.254828.086929l-4.04836-4.019704c-.333805-.369084-.304418-.938178.065638-1.271107.339217-.305184.84694-.305988 1.18632-.019713L11.105 15.047l.000763-8.347c0-.458821.343336-.837452.787106-.892988L12.005763 5.8l-.112894.007012z"/></svg>
```

Comment:
For some reason, this doesn't display, can you take a look?

Reply:
Fixed.